examples/lineChart.html

```diff
@@ -29,6 +29,8 @@
 <div style="position:absolute; top: 0; left: 0;">
     <button onclick="randomizeFillOpacity();">Randomize fill opacity</button>
     <button onclick="expandLegend();">Expand/Contract Legend</button>
+    <button onclick="toggleLegend();">Toggle Legend Top/Bottom</button>
```

Comment:
This is going to make the buttons overlap on the legend when it's at the top, at least in the default layout where it's in an iframe with a set width... but that's an issue that's beyond the scope of this PR so it's fine for now.